src/docs/product/alerts-notifications/index.mdx

```diff
@@ -8,19 +8,40 @@ description: "Sentry categorizes notifications into alerts and other types of no
 
 Sentry categorizes notifications broadly into alerts and non-alert notifications.
 
-## Alerts
+## Error Alerting
 
-[Alerts](/product/alerts-notifications/alerts/) notify you about problems in your application and can be sent to many supported integrations. You can set alerts on both errors and performance data.
+Alerts notify you about problems in your application and can be sent to many supported integrations. You can set alerts on both errors and performance data.
 
-- **Errors**: Sentry automatically groups individual error events into Issues based on stack trace and other factors. You can then set alerts on changes in those issues (such as new issues or issues affecting many users), or on the raw events (such as frequency of events with a given tag).
+### Issue Alerts
 
-- **Performance**: You can alert on performance metrics like latency, apdex, failure rate, and throughput.
+Sentry automatically groups individual error events into issues based on stack trace and other factors. You can then set alerts on changes in those issues (such as new issues or issues affecting many users), or on the raw events (such as frequency of events with a given tag).
 
-For more details, see the [full documentation on Alerts](/product/alerts-notifications/alerts/).
+For more details, see the [full documentation on Issue Alerts](/product/alerts-notifications/alerts/#issue-alerts).
 
-## Non-Alert Notifications
+### Metric Alerts
 
-[Non-alert notifications](/product/alerts-notifications/notifications/) are all other types of Sentry notifications such as:
+Use metric alerts on error data to compute the number of events or users affected by events in your project. A metric is the value of an aggregate function like `count()` or `avg()`. You can filter events by attributes and tags, which is particularly useful for aggregating across events that aren't grouped into single issues.
```

Comment:
Perhaps start with clear definition of metric alerts: "Metric alerts are an aggregate of a function, such as `count()` or `avg()`. They function in a distinctly different fashion from issue alerts, which focus on individual issues." Might also be good to provide an example of each on this intro page.